contrib/devtools/gen-manpages.py

```diff
@@ -57,6 +58,30 @@
     print('To properly generate man pages, please commit your changes (or discard them), rebuild, then run this script again.')
     print()
 
+# Check enabled build options
+config = configparser.ConfigParser()
+conffile = os.path.join(builddir, 'test/config.ini')
+config.read(conffile)
+
+error_message = "Aborting generating manpages…\nError: {component} support is not enabled.\nPlease enable it and try again."
+
+if not config['components'].getboolean('HAVE_SYSTEM'):
```

Comment:
This seems arbitrary and prone to breakage, giving the hardcoding of specific components. It also doesn't cover all of the relevant components in any case, i.e at least decl_fork and natpmp (which both effect manpage output) are missing any sort of check.

Reply:
Not sure. Although I think any changes here can wait until after CMake, as this script will likely need to be (partially) rewritten after that change in any case.

Reply:
This is hardcoding component names / variables, and they are likely to change during the switchover.